dev-docs/source/VSCode.rst

```diff
@@ -186,7 +190,36 @@ you will want to make your file look something a little like this:
         ]
     }
 
-**Window:**
+*Workbench*
+
+To debug C++ and start directly into the Workbench, add this to the configuration list in ``launch.json``.
+
+.. code-block:: javascript
+
+    {
+      "name": "(gdb) Workbench C++ Only",
+      "type": "cppdbg",
+      "request": "launch",
+      "program": "/usr/bin/python2.7", // Path to your used Python interpreter, here and below
+      "args": ["Path/To/Build/Directory/bin/workbench", "&&","gdb","/usr/bin/python2.7","$!"], // $! gets the process ID
+      "stopAtEntry": false,
+      "cwd": "Path/To/Build/Directory/bin", // this should point to bin inside the build directory
+      "environment": [],
+      "externalConsole": false,
+      "MIMode": "gdb",
+      "preLaunchTask": "Build Mantid",
```

Comment:
Probably best to remove the preLaunchTask unless they actually want, some people won't.
```suggestion

```

Reply:
I've left it in so it matches the MantidPlot section, I think people can remove it from both if they don't want to do that